charts/gitops-server/values.yaml

```diff
@@ -11,48 +11,58 @@ image:
 imagePullSecrets: []
```

Comment:
Can you add a comment @ the start of this file explaining what the `# --` format means, please?